source/actors/player/HeartContainer.hx

```diff
@@ -0,0 +1,97 @@
+package actors.player;
+
+import flixel.FlxSprite;
+import flixel.graphics.FlxGraphic;
+
+class HeartContainer extends FlxSprite {
+
+    private static var FULLHEART:String = "fullheart";
+    private static var THREEQUARTERHEART:String = "threequarterheart";
+    private static var HALFHEART:String = "halfheart";
+    private static var ONEQUARTERHEART:String = "onequarterheart";
+    private static var EMPTYHEART:String = "emptyheart";
+    public static var MAXFRAGAMOUNT:Int = 4;
```

Comment:
For static variables, let's use the UPPER_CASE_SEPARATED_BY_UNDERSCORES style, makes it a bit more readable over having all of the letters jammed together.